### PR TITLE
Fixed broken report links

### LINKED ISSUE #2804: Test Reports Not Accessible
If you run all the tests (from localhost or on Travis), the report files are not generated properly - there are broken links. 
Also "Authorization tests" should be moved from "Functional tests" to "integration tests"

**Testflow:**
1. Open the Kapua project (develop branch)
2. Run command `mvn clean install -DskipTests`
3. Run command `mvn verify -Dcommons.db.schema=kapuadb -Dcommons.settings.hotswap=true -Dbroker.host=localhost -Dgroups='!org.eclipse.kapua.qa.markers.junit.JUnitTests' -Dcucumber.options="--tags ~@rest"` (skip the Unit and the Rest tests)
4. Go to Reports folder in QA and try to open the main "index.html" file and click through the reports - some of them are not accessible.

**Expected behavior:**
When you run all the tests, all the links in the test reports should be available.

**Screenshots**
N/A

**Version of Kapua**
1.1.0-SNAPSHOT

**Type of deployment**
[ ] Local Vagrant deployment
[x] Docker
[ ] Openshift (in its variants) 
[ ] Others

**Main component affected**
[x] Console (in case of console please report info on which browser you encountered the problem)
[ ] REST API
[ ] Message Broker
[ ] - Others

Browsers used: 
- Google Chrome (Version 72.0.3626.119 (Official Build) (64-bit))
- Safari (Version 12.0.3 (14606.4.5))
- Mozilla Firefox (65.0.1 (64-bit))
- Opera (Version:58.0.3135.79)

### PR DESCRIPTION
**Pull request by ct-ajovanovic**
Brief description of the PR.
Fixed broken report links.

**Related Issue**
This PR fixes/closes #2804 

**Description of the solution adopted**
Updated `kapua-cucumber-report.xml` and `index.html` files from `dev-tools` module to fix the broken links. Updated runner classes for `jobEngineService` tests and added new links to display the results.

**Screenshots**
_None_

**Any side note on the changes made**
This PR is based on the work done in PR #2801, and it should be merged after it.

### CODE REVIEW

**Comment by LeoNerdoG** [comment]
hey @Coduz,
I have checked this PR and it looks OK. 
After the PR #2801 is merged and tests complete on this one - can you please merge it?

**Comment by ct-ajovanovic** [comment]
Hi @Coduz @LeoNerdoG 
This PR was rebased to the latest develop, after the merge of #2801 .
It can be merged when the tests pass.

**Review by Coduz — APPROVED** [review]

**Pull request merged**
